app/models/concerns/reviewable.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Reviewable
+  extend ActiveSupport::Concern
+
+  included do
+    has_one :review, as: :reviewable, dependent: :destroy
```

Comment:
Not for this PR but what should happen if there are multiple rounds of reviews where the conditions are sent back for amendments by the reviewer? We should probably track that history (especially for minors/majors) so we might need to change this to a `has_many` and have a setup where we only append new reviews and show the latest.

Reply:
Agreed, needs to be a `has_many` reviews as there can be multiple rounds

Reply:
Agree, but as I didn't want to expand it more I've left it as `has_one :review` which is what is needed for this